### ISSUE TITLE
[Cheatsheets] Add POELab information

### ISSUE THREAD

**Issue by domialex** [domialex/Sidekick#345]
POELab seems to follow a very simple url for the images.
`https://www.poelab.com/wp-content/labfiles/yyyy-mm-dd_{type}.jpg` where type is `uber`, `merciless`, `cruel` and `normal`.
Let's not forget to add credit where it's due.

**Comment by Newbrict** [comment]
great idea

**Comment by Amp0** [comment]
I look a bit at this feature.
There is one issue at the moment, the url of the image contains a unique id that looks randomly generated.
Example : `https://www.poelab.com/wp-content/labfiles/2020-09-17_000052119_uber.jpg`

A possible solution to this would be to scrape the image url from the website.

I also realised there is a lab compass file on the page (which sadly has the same random id)
Example :  `https://www.poelab.com/wp-content/labfiles/uber-2020-09-17_000052119.json`

I played a bit with it and started to implement a graphical representation using the file, I think it would be great to have a custom made labyrinth layout image using this file rather than only showing the poelab one.
![image](https://user-images.githubusercontent.com/13069770/93476726-66e4bb00-f8fa-11ea-8937-1fc51e3d8d4e.png)

The image above is just a simple test I made quickly by reading the json file.
